### DISCUSSION TITLE
Upgrade mio version

### DISCUSSION THREAD

**Discussion by wngr** [tokio-rs/tokio#3566] [General] [answered]
Mio recently saw a couple of (IMO) critical bug fixes, where accepting on TCP socket on x86 Android is broken in mio up to version 0.7.8. Because of that, a version 0.7.9 was released (see https://github.com/tokio-rs/mio/issues/1462). It would be great to upgrade to mio 0.7.9 to get those fixes upstream into tokio.

I'm ignorant how this project handles those issues, so starting a discussion here first.

Thanks!

**Comment by Darksonn** [comment, marked as answer]
Tokio's current dependency on mio is "any version 0.7.x with x ≥ 6", so you should already be able to use mio version 0.7.9 with Tokio.

**Reply by wngr** [reply]
Thanks :-). So  I seems I was ignorant to Cargo's dependency behavior after all.
